.circleci/config.yml

```diff
@@ -127,6 +127,15 @@ jobs:
           name: Lint Go
           command: 'make lint-go'
 
+  shellcheck:
+    machine: true
```

Comment:
Here, I wanted to use the same command make command as it intendant to be used locally